### PR TITLE
Compress training data

### PR DESCRIPTION
**Pull request by dimitrismistriotis**
On writing these lines from <https://pypistats.org/packages/alt-profanity-check>:

```
Downloads last day: 828
Downloads last week: 7,826
Downloads last month: 187,159
```

We have seen more extending the 200K downloads per month. Additionally, there is a fork which has removed the data probably to install the library faster.

It is a requirement and the reason that this project exists that data should be "as close as possible" to the models being trained. On the other hand 99.99% of people using the library want to ... use the library and are not interested into having access to the data. For them the download of an approximately 60MB file is not helpful.

With that we (me + @menkotoglou) discussed two possibilities: (a) move the data to another repository, (b) compress them. Opted for (b)

Last step was the choice of the algorithm, long story short decided on 7z, knowing that some systems ship without it installed:

image

```
ls -lh clean_data.*
-rw-rw-r-- 1 dimitrios dimitrios 64M Jun 13 09:16 clean_data.csv
-rw-rw-r-- 1 dimitrios dimitrios 18M Jun 13 09:40 clean_data.tar.xz
-rw-rw-r-- 1 dimitrios dimitrios 26M Jun 13 09:42 clean_data.zip
```

### CODE REVIEW

File: profanity_check/data/decompress_data [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+OUTPUT_FILE=clean_data.csv
+
+echo ""
+echo "Decompress clean_data.csv"
+echo ""
+
+#
+# Check that 7z is available, source: https://stackoverflow.com/a/677212
+#
+command -v 7z >/dev/null 2>&1 || { echo >&2 "I require 7z but it's not installed.  Aborting."; exit 1; }
```
Comment (menkotoglou) [inline profanity_check/data/decompress_data:23]: Minor but two spaces after .
```suggestion
command -v 7z >/dev/null 2>&1 || { echo >&2 "I require 7z but it's not installed. Aborting."; exit 1; }
```

**Review by menkotoglou — APPROVED** [review]
Just one minor comment, LGTM other than that! :clap:

File: profanity_check/data/decompress_data (hunk 1 above)
Comment (dimitrismistriotis) [reply profanity_check/data/decompress_data]: Clicked to add change

**Review by menkotoglou — APPROVED** [review]
:100:

**Pull request merged**